source/tests/universal/common/cases/atomic_model/ener_model.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+
+from .utils import (
+    AtomicModelTestCase,
+)
+
+
+class EnerAtomicModelTest(AtomicModelTestCase):
```

Comment:
How does one want to add a new test like one for permutation in-variance of the energy? It may not be a good idea to derive from an `AtomicModelTestCase` although the name somehow indicating the derivation.

Reply:
Why cannot one add to this class?

Reply:
different test are better provided in different test cases. I do not want to test e.g. autodiff when I am testing translational invariance.

Reply:
It's unexpected that they will run together. Each test in a test case should run individually.

Reply:
Do you mean you want to select a certain test? `-k` can be used to do so.

Reply:
by your definition of the base class , all the test will run